aiida/cmdline/commands/group.py

```diff
@@ -227,48 +245,46 @@ def group_list(all_users, user_email, type, with_description, count,
     from aiida.orm.group import get_group_type_mapping
     from aiida.orm.backend import construct_backend
     from tabulate import tabulate
+    from aiida.common.extendeddicts import AttributeDict
+
+    kwargs = AttributeDict(kwargs)
```

Comment:
this is a really weird construction it seems. Why not keep the explicit function signature (but get rid of *args), and then you can just those argument as variables every where instead of attributes of `kwargs`, and you can also get rid of all the `# pylint: disable=no-member` statements.